src/prasad/CodingExam1.java

```diff
@@ -0,0 +1,55 @@
+package prasad;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class CodingExam1 {
+	String uName, fname, lName, temp;
+	char fch;
+	int count;
+
+	void validateID(WebDriver driver) {
+		driver.findElement(By.linkText("Demo Tables")).click();
+		List<WebElement> columns = driver.findElements(By.xpath("//table[@id= 'table1']/thead/tr/th"));
+		int noOfCol = columns.size();
+
+		List<WebElement> rows = driver.findElements(By.xpath("//table[@id= 'table1']/tbody/tr"));
+		for (int rowIndex = 1; rowIndex <= rows.size(); rowIndex++) {
+			for (int colIndex = 2; colIndex <= noOfCol; colIndex++) {
```

Comment:
This is not good approach for data comparing.
![image](https://user-images.githubusercontent.com/58640315/105631101-ac019680-5e72-11eb-86a4-614ed6acbe80.png)